run_test.go

```diff
@@ -71,13 +70,13 @@ func TestCage_Run(t *testing.T) {
 		)
 		cagecli := cage.NewCage(&cage.Input{
 			Env:  env,
-			ECS:  ecsMock,
+			Ecs:  ecsMock,
 			Time: test.NewFakeTime(),
 		})
 		result, err := cagecli.Run(ctx, &cage.RunInput{
 			Container: &container,
 			Overrides: overrides,
-			MaxWait:   1,
+			// MaxWait:   1,
```

Comment:
I apologize for bringing this up after approval, but could this possibly be something that was forgotten to be deleted?